### PR TITLE
Remove duplicate Prod and SparseToDense ops from converter

### LINKED ISSUE #7648: Op Collision in OperationMapper
[OperationMapper](https://github.com/tensorflow/tfjs/blob/master/tfjs-converter/src/operations/operation_mapper.ts#L45) stores a map of all the TFJS ops in the `opMappers` parameter. It generates this map by iterating all the ops in [op_list/](https://github.com/tensorflow/tfjs/tree/master/tfjs-converter/python/tensorflowjs/op_list) and uses `tfOpName` as the key. This key is not unique between ops in different categories.

There are two collisions in the current set of supported ops:
* [basic_math/Prod](https://github.com/tensorflow/tfjs/blob/master/tfjs-converter/python/tensorflowjs/op_list/basic_math.json#L800) and [reduction/Prod](https://github.com/tensorflow/tfjs/blob/master/tfjs-converter/python/tensorflowjs/op_list/reduction.json#L222)
  * The signature for reduction/Prod appears to be a subset of the signature for basic_math/Prod.
* [slice_join/SparseToDense](https://github.com/tensorflow/tfjs/blob/master/tfjs-converter/python/tensorflowjs/op_list/slice_join.json#L364) and [normalization/SparseToDense](https://github.com/tensorflow/tfjs/blob/master/tfjs-converter/python/tensorflowjs/op_list/normalization.json#L22)
  * The signatures between these two appear identical.

### PR DESCRIPTION
**Pull request by mattsoulanille**
Fixes #7648 

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

### CODE REVIEW

File: tfjs-converter/python/tensorflowjs/op_list/reduction.json [hunk 1]
```diff
@@ -238,6 +238,12 @@
         "tfName": "keep_dims",
         "name": "keepDims",
         "type": "bool"
+      },
+      {
+        "tfName": "T",
+        "name": "dtype",
+        "type": "dtype",
+        "notSupported": true
```
Comment (mattsoulanille) [inline tfjs-converter/python/tensorflowjs/op_list/reduction.json:246]: @pyu10055 do you know what this 'T' attribute is for?

**Review by pyu10055 — APPROVED** [review]

File: tfjs-converter/src/operations/executors/basic_math_executor.ts [hunk 2]
```diff
@@ -159,10 +159,6 @@ export const executeOp: InternalOpExecutor =
               getParamValue('x', node, tensorMap, context) as Tensor)];
         case 'Rsqrt':
           return [ops.rsqrt(getTensor(node.inputNames[0], tensorMap, context))];
-        case 'Prod':
```
Comment (chunnienc) [inline tfjs-converter/src/operations/executors/basic_math_executor.ts:162]: For removing existing executor: are you sure that there are no old models converted with category alias?

**Review by chunnienc — APPROVED** [review]

File: tfjs-converter/src/operations/executors/basic_math_executor.ts (hunk 2 above)
Comment (mattsoulanille) [reply tfjs-converter/src/operations/executors/basic_math_executor.ts:162]: I checked an example model file, and we don't actually save the category of an op in the model.json. I don't think this should affect our model support, but good idea to check.

Comment (chunnienc) [reply tfjs-converter/src/operations/executors/basic_math_executor.ts:162]: yeah `model.json` does not store the category name, my bad. Agreed that it shouldn't affected converted models.

**Pull request merged**